version.txt

```diff
@@ -1 +1 @@
-2.11.0
```

Comment:
nit: Can you add 
```
# Format:
# module:released-version:current-version
```

like it is in: https://github.com/googleapis/gax-java/blob/main/versions.txt

Reply:
Done